target/linux/ath79/image/generic.mk

```diff
@@ -1398,6 +1398,16 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/jjplus_jwap230
+  SOC := qca9558
+  DEVICE_VENDOR := jjPlus
+  DEVICE_MODEL := JWAP230
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-kernel | pad-to $$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size 16000k
```

Comment:
Please wrap after 74 chars. max.

Please specify IMAGE_SIZE variable and use check-size without argument then.

Reply:
Let me know if I got it right, please ?